src/datapilot/core/platforms/dbt/cli/cli.py

```diff
@@ -35,15 +35,24 @@ def dbt():
     required=False,
     help="Path to the DBT config file",
 )
-def project_health(manifest_path, catalog_path, config_path=None):
+@click.option(
+    "--select",
+    required=False,
+    default=None,
+    help="Selective model testing. Specify one or more models to run tests on.",
```

Comment:
can may be add documentation/example for this 
think we should just support following way https://docs.getdbt.com/reference/node-selection/syntax
Also I think this should be space separated simialr how dbt approaches it 
```

The --select flag accepts one or more arguments. Each argument can be one of:

a package name
a model name
a fully-qualified path to a directory of models
a selection method (path:, tag:, config:, test_type:, test_name:)
```